### DISCUSSION TITLE
get some corrupt file after turn off verify ssl

### DISCUSSION THREAD

**Discussion by farrostaufiqur** [Ljzd-PRO/KToolBox#21] [Q&A]
here the result of some result

5f468f7f-8aef-48ab-b2fa-0818f677f947.jpg: 19.0kB [00:00, 256kB/s]
a4971091-da4f-4f2a-9340-e75b985c9316.jpg: 19.0kB [00:00, 301kB/s]
d5a3d489-fef2-4907-a288-bec1d37af304.jpg: 19.0kB [00:00, 223kB/s]
af28dc83-86a1-4c3b-9c53-e0c06ff8f0ae.jpg: 19.0kB [00:00, 365kB/s]                 | 16.4k/4.43M [00:00<01:07, 65.0kB/s]
28d0fb14-ed5c-4958-9a62-f3d707547d2e.jpg: 19.0kB [00:00, 395kB/s]
1449101_de2e3f5a-838f-4b88-a061-95725443fda3.jpg: 19.0kB [00:00, 279kB/s]         | 65.5k/4.43M [00:02<02:18, 31.5kB/s]
3540852b-75a6-4e33-8f8c-61ef50e3df21.jpg: 100%|████████████████████████████████████| 4.43M/4.43M [00:09<00:00, 445kB/s]

all 19kb file is corrupt

**Comment by Ljzd-PRO** [comment]
Try to open all 19KB files in text file format, there may be error messages inside, as their sizes are consistent.

**Comment by Ljzd-PRO** [comment]
I tried downloading posts without turning off `ssl_verify` yesterday, and it succeeded

**Comment by Ljzd-PRO** [comment]
Perhaps it's related to DNS resolution? It's possible that DNS resolved the domain of the download link to an incorrect IP, resulting in certificate verification failure. You can check if you can download the original image files manually after entering the Kemono site in your browser

**Reply by Ljzd-PRO** [reply]
If the download of the final 4.43MB file was successful and not corrupted, then DNS may not contribute to it.

**Comment by farrostaufiqur** [comment]
based off coom-dl discord, i think its because kemono/coomer just improve their web security
